### PR TITLE
Limit FIPS functional testing on OpenJcePlusTests and security

### PR DESCRIPTION
**Pull request by llxia**
related: backlog/issues/1494

### CODE REVIEW

**Comment by llxia** [comment]
[internal run](https://hyc-runtimes-jenkins.swg-devops.com/view/Test_grinder/job/AQA_Test_Pipeline_Release/75/) => cancel the job after the child jobs were launched as expected

- [FIPS child job](https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk11_j9_extended.functional_ppc64le_linux_fips140_3_openjceplusfips/4/):
image

- [Regular child job](https://hyc-runtimes-jenkins.swg-devops.com/job/Test_openjdk11_j9_sanity.functional_aarch64_linux/682/):
image

File: buildenv/jenkins/aqaTestPipeline.groovy [hunk 1]
```diff
@@ -150,16 +150,23 @@ def generateJobs(jobJdkVersion, jobTestFlag, jobPlatforms, jobTargets) {
...
                 } else if (TARGET.contains('functional')) {
-                    VENDOR_TEST_REPOS = 'git@github.ibm.com:runtimes/test.git'
-                    VENDOR_TEST_BRANCHES = params.ADOPTOPENJDK_BRANCH ?: 'master'
-                    VENDOR_TEST_DIRS = 'functional'
+                    if (jobTestFlag.contains("FIPS")) {
+                        if (!buildList) {
+                            buildList = "functional/OpenJcePlusTests,functional/security"
+                        }
+                    } else {
+                        VENDOR_TEST_REPOS = 'git@github.ibm.com:runtimes/test.git'
```
Comment (pshipton) [inline buildenv/jenkins/aqaTestPipeline.groovy:166]: These things don't need to be set when running the FIPS buildList?

**Review by LongyuZhang — APPROVED** [review]
LGTM

File: buildenv/jenkins/aqaTestPipeline.groovy (hunk 1 above)
Comment (llxia) [reply buildenv/jenkins/aqaTestPipeline.groovy:166]: For FIPS, the decision is to run tests under `OpenJcePlusTests` and `security` folder only.

**Review by pshipton — APPROVED** [review]

**Pull request merged**